plugins/env/lib/samson_env/samson_plugin.rb

```diff
@@ -75,30 +79,30 @@ def write_dotenv(base_file, groups)
 # TODO: make a edit page and link to that
 Samson::Hooks.callback(:link_parts_for_resource) do
   [
-    "EnvironmentVariable",
-    ->(env) do
-      scope = " for #{env.scope.name}" if env.scope
-      parent = " on #{env.parent.name}" if env.parent
-      ["#{env.name}#{scope}#{parent}", EnvironmentVariable]
-    end
+      "EnvironmentVariable",
```

Comment:
Extra indents? You might have incorrect spacing settings on RubyMine. They should be set to 2 spaces for each level of indentation.

Reply:
done